### PR TITLE
Modify the UI for attribute mapping summary

### PR DESCRIPTION
**Pull request by PoulainMaxime**

### CODE REVIEW

**Review by elisegriset92 — COMMENTED** [review]
Need linter and maybe review from @amaury-hanser ?

File: _dev/src/components/product-feed/commons/table-row-mapping.vue [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
-    <b-td class="pb-0 align-top pt-md-0 pb-md-1">
+    <b-td class="align-top">
       <div class="container-mapping-summary">
         <span
-          class="mb-0 text-left ellipsis-mapping-summary-text"
-          :class="{'text-secondary': attributeNotMapped}"
+          class="mb-0 text-right ellipsis-mapping-summary-text"
+          :class="{'text-danger-light': attributeNotMapped}"
         >
+          <i class="material-icons-outlined align-items-with-text" v-if="attributeNotMapped">
```
Comment (Quetzacoalt91) [inline _dev/src/components/product-feed/commons/table-row-mapping.vue:16]: Linter reported some issues to take in account

File: _dev/src/assets/scss/layouts/_table.scss [hunk 2]
```diff
@@ -35,13 +35,26 @@
...
+}
+
 .ellipsis-mapping-summary-text {
   display: block;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
 }
 
+.table-border-bottom {
```
Comment (amaury-hanser) [inline _dev/src/assets/scss/layouts/_table.scss:54]: Is it different from the class `border-bottom` ?

**Review by Quetzacoalt91 — DISMISSED** [review]

File: _dev/src/components/product-feed/settings/summary/summary.vue [hunk 3]
```diff
@@ -72,33 +72,73 @@
...
+                          src="@/assets/images/google-icon-grey.svg"
+                          width="20"
+                          height="20"
+                        >
+                        <p>
+                          {{ $t('productFeedSettings.summary.tableHeader1') }}
+                        </p>
+                      </div>
+                      <div class="d-inline-block text-right no-icon-mapping">
+                        <img
```
Comment (amaury-hanser) [inline _dev/src/components/product-feed/settings/summary/summary.vue:115]: Should be a `background-image` on the `tr`

**Review by amaury-hanser — CHANGES_REQUESTED** [review]
Som qui changes, moving rule in correct file and use the arrow icon in a `background-image`

**Review by amaury-hanser — APPROVED** [review]

**Pull request merged**